rs/universal_canister/lib/build.rs

```diff
@@ -0,0 +1,8 @@
+fn main() {
+    // lib.rs depends on the environment variable UNIVERSAL_CANISTER_WASM_PATH which is set to the right value in BUILD.bazel.
+    // To make cargo clippy / build work we set the variable to a fake value.
```

Comment:
Would [`option_env!`](https://doc.rust-lang.org/std/macro.option_env.html) work instead?

Reply:
I'm not sure since that results in an Option<Path> while `include_bytes!` expects a Path.

Reply:
I think you can `.unwrap()` it in rust 1.83 (we're almost there 🙃 )

Reply:
Yeah we can simplify it once we upgrade to 1.83.